src/main/java/control/ControllerManager.java

```diff
@@ -18,6 +18,9 @@
     private TimelineController timelineControl;
 
     @Getter
+    private ToolViewController toolViewController;
```

Comment:
Could you also add this controller to the overloaded constructor to keep things consistent. This overloaded constructor is, among others, usefull for testing.